script/test

```diff
@@ -110,9 +110,10 @@ do_build()
         "-DCMAKE_INSTALL_PREFIX=/usr"
         "-DOT_THREAD_VERSION=1.3"
         "-DOTBR_DBUS=ON"
-        "-DOTBR_WEB=ON"
-        "-DOTBR_UNSECURE_JOIN=ON"
+        "-DOTBR_REGION_FILE=/tmp/otbr-region-file"
```

Comment:
May not be a good idea to use `/tmp/` for persistent information.

Why not use OT settings?

Reply:
The intended use case is recover the region code after crash. So I think `/tmp` is good enough.